### ISSUE TITLE
Add gradient clipping in Classifier._train_model()

### ISSUE THREAD

**Issue by bhancock8** [HazyResearch/metal#136]
Perform clipping (optionally) right after loss.backward(). Be sure to add a unit test that it's working by checking actual gradient values.

**Comment by vincentschen** [comment]
The latest version of the classifier has been implemented in the [Snorkel Classification](https://snorkel.readthedocs.io/en/v0.9.0/packages/classification.html) package. Please make relevant feature requests in this repository!
